content/java/concepts/math-methods/terms/log1p/log1p.md

```diff
@@ -0,0 +1,71 @@
+---
+Title: '.log1p()'
+Description: 'Returns the natural logarithm (base e) of the sum of a double value as a parameter and 1.'
+Subjects:
+  - 'Computer Science'
+Tags:
+  - 'Methods'
+  - 'Arithmetic'
+CatalogContent:
+  - 'learn-java'
+  - 'paths/computer-science'
+---
+
+The **`.log1p()`** method returns `ln (a + 1)`, the natural logarithm of a + 1.
+
+## Syntax
+
+```pseudo
+Math.log1p(a);
+```
+
+- The `.log1p()` method takes one parameter `a`.
+- If `a` is `NaN`, then return value is `NaN`.
```

Comment:
Lines 23-27 should be the same as the ones in the entry for the previous two functions.